### PR TITLE
feat (VSCode) : Add tasks to build current project 

### PR DESCRIPTION
**Pull request by ShreyasJejurkar**
PR adds the ability to build the current project provided that any `*.cs` file in that project is open in the editor.
I have taken this approach from dotnet/Roslyn repo and did some modifications to it to suit the needs! 

How to build a project 
Consider you are editing some .cs file and you wanted to build that project only. Just press `CTRL + SHIFT + P` to command pallet search for `Run Task` command. Then select `Build current project`. And now it will start building the project. 

Note - Please review this commit https://github.com/dotnet/aspnetcore/commit/811c9d8b49a8e2fe54e4f47bdcadb0be4ffedaf4 , ignore other previous commits those are already merged to main, but not sure why those are getting shown here! 

https://user-images.githubusercontent.com/17148381/147631936-37a84aae-b53c-491e-b41a-1062ed0bc8dd.mp4

cc @captainsafia

### CODE REVIEW

**Comment by mkArtakMSFT** [comment]
Thanks for your PR, @ShreyasJejurkar.
@captainsafia could you please review this? Thanks!

File: .vscode/tasks.json [hunk 1]
```diff
@@ -75,6 +75,22 @@
         "panel": "new"
       },
       "problemMatcher": []
+    },
+    {
+      "label": "Build current project",
+      "type": "shell",
+      "command": "pwsh",
+      "args": [
+        "${workspaceFolder}/eng/scripts/vscode-build.ps1",
```
Comment (captainsafia) [inline .vscode/tasks.json:84]: Can we get a bash variant for this so that this task works in a Codespaces environment?

Comment (ShreyasJejurkar) [reply .vscode/tasks.json:84]: I did this PR in codespaces only. And interestingly it worked there. Seems pwsh (PowerShell) is pre-installed there!

Comment (dougbu) [reply .vscode/tasks.json:84]: This makes me worry a bit but I'm more concerned about developers building directly on non-Windows platforms. Can't assume `pwsh` will be available for everyone, even if the Codespaces environment includes it.

Comment (ShreyasJejurkar) [reply .vscode/tasks.json:84]: yeah, I can understand! I will try to add a bash variant as well!

**Comment by ShreyasJejurkar** [comment]
> Hi @ShreyasJejurkar. It seems you haven't touched this PR for the last two weeks. To avoid accumulating old PRs, we're marking it as `stale`. As a result, it will be closed if no further activity occurs **within 4 days of this comment**. You can learn more about our Issue Management Policies [here](https://aka.ms/aspnet/issue-policies).

keep for some time. I will get this done.

**Pull request closed without merging**